### ISSUE TITLE
must write jsDoc at properties for interface and public Construct

### ISSUE THREAD

**Issue by ren-yamanashi** [ren-yamanashi/eslint-cdk-plugin#98]
### Feature Description

Add a rule that jsDoc must be attached to properties to be exposed in interface and Construct.

### Motivation

With the jsDoc written, it becomes clearer what the properties exposed in the interface and Construct mean.
This increases the readability of the code.

### Invalid Code Examples

```typescript
interface MyConstructProps {
 // ❌ must write jsDoc
  readonly bucket: IBucket
}
```

### Valid Code Examples

```typescript
interface MyConstructProps {
 /**
  * S3 bucket to be specified for the resource
  */
  readonly bucket: IBucket
}
```

### Rule Type

problem (possible errors)

### Implementation Details

- [ ] This rule is related to AWS CDK best practices
- [x] I'm willing to submit a PR for this feature

### Additional Context

Similar rules exist for awslint

https://github.com/aws/aws-cdk/blob/main/packages/awslint/lib/rules/docs.ts#L35-L47
